internal/provider/data_source_policy_set_test.go

```diff
@@ -43,6 +43,61 @@ func TestAccTFEPolicySetDataSource_basic(t *testing.T) {
 						"data.tfe_policy_set.bar", "organization", org.Name),
 					resource.TestCheckResourceAttr(
 						"data.tfe_policy_set.bar", "policy_ids.#", "1"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_policy_set.bar", "agent_enabled", "false"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_policy_set.bar", "workspace_ids.#", "1"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_policy_set.bar", "excluded_workspace_ids.#", "1"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_policy_set.bar", "project_ids.#", "1"),
+					resource.TestCheckResourceAttr(
+						"data.tfe_policy_set.bar", "vcs_repo.#", "0"),
+				),
+			},
+		},
+	},
+	)
+}
+
+func TestAccTFEPolicySetDataSource_pinnedPolicyRuntimeVersion(t *testing.T) {
+	skipUnlessBeta(t)
+	tfeClient, err := getClientUsingEnv()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	sha := genSentinelSha(t, "secret", "data")
+	version := genSafeRandomSentinelVersion()
+
+	org, orgCleanup := createBusinessOrganization(t, tfeClient)
+	t.Cleanup(orgCleanup)
```

Comment:
If this is available in TFE, we should add a check here to only upgrade the subscription if it is not `enterpriseEnabled()`

Reply:
isn't that check already included inside the createBusinessOrg function here? https://github.com/hashicorp/terraform-provider-tfe/blob/main/internal/provider/testing.go#L76

Reply:
`createBusinessOrg` skips the test for TFE since there are no feature sets in TFE. So it should look something along these lines:

```go
var org *tfe.Organization
var cleanup func()

if enterpriseEnabled(t) {
  org, cleanup = createOrganization()
} else {
  // we are in TFC so we have a subscription to upgrade, otherwise nothing to upgrade
  org, cleanup = createBusinessOrganization()
}
```